solr/core/src/test-files/solr/collection1/conf/solrconfig-memory-circuitbreaker.xml

```diff
@@ -78,11 +78,12 @@
 
   </query>
 
-  <circuitBreaker class="solr.CircuitBreakerManager" enabled="true">
-    <str name="memEnabled">true</str>
-    <str name="memThreshold">75</str>
-    <str name="cpuEnabled">true</str>
-    <str name="cpuThreshold">75</str>
+  <circuitBreaker class="org.apache.solr.util.circuitbreaker.MemoryCircuitBreaker" enable="true">
+    <int  name="threshold">75</int>
+  </circuitBreaker>
+
+  <circuitBreaker class="org.apache.solr.util.circuitbreaker.CPUCircuitBreaker" enable="true">
+    <int  name="threshold">75</int>
```

Comment:
Not sure if the `int` might need to be non-`int` here to match the `setThreshold(double threshold)` signature e.g.
```suggestion
    <double  name="threshold">75</double>
```